### ISSUE TITLE
signup is broken

### ISSUE THREAD

**Issue by iamdey** [CaliOpen/Caliopen#286]
ERR 401 on POST

**Comment by iamdey** [comment]
it comes from the api:

> POST http://localhost:31415/api/v1/users
>
> {
>	"username": "dey",
>	"password": "123456",
>	"recovery_email": "dey@caliopen.local"
> }

```
apiv1_1            | 2017-05-03 18:42:55,970 INFO  [caliopen_main.user.core.user][waitress] 'NoneType' object has no attribute 'family_name'
```

```
{
	"errors": [
		{
			"code": 401,
			"description": "Wrong credentials (e.g., bad password or username)",
			"component": "collection_User",
			"values": 401,
			"message": "'NoneType' object has no attribute 'family_name'",
			"property": "AuthenticationError",
			"type": "Authentication error"
		}
	]
}
```

**Comment by sapiens-sapide** [comment]
Fix by commit #B7D7457

**Comment by iamdey** [comment]
no # before commit hash @sapiens-sapide to make auto-link: B7D7457
